USAGE.md

```diff
@@ -44,33 +42,34 @@ kubectl patch svc -n ingress-nginx ingress-nginx-controller --type='json' -p='[{
 
 # remove --publish-status-address=localhost, don't set loadbalancer status to localhost (will use IP of ingress)
 kubectl patch deployment -n ingress-nginx ingress-nginx-controller --type='json' -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/args/9"}]'
+```
 
+## 5. Enable SSL passthrough
+```bash
 # enable ssl passthrough
 kubectl patch deployment -n ingress-nginx ingress-nginx-controller --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value":"--enable-ssl-passthrough"}]'
 ```
 
-## 5. Deploy Vault
+## 6. Deploy Vault
 Run the following commands to deploy Vault:
 ```bash
 cd Vault-Deployment
 terraform init
 terraform apply
 ```
 
-## 6. Unseal Vault
+## 7. Unseal Vault
 Run the following commands to unseal Vault:
 ```bash
 kubectl exec -n vault vault-0 -- vault operator init -key-shares=1 -key-threshold=1 -format=json > ./cluster-keys.json
 kubectl exec -n vault vault-0 -- vault operator unseal $(cat ./cluster-keys.json | jq -r ".unseal_keys_b64[]")
-kubectl exec -n vault vault-1 -- vault operator unseal $(cat ./cluster-keys.json | jq -r ".unseal_keys_b64[]")
-kubectl exec -n vault vault-2 -- vault operator unseal $(cat ./cluster-keys.json | jq -r ".unseal_keys_b64[]")
 ```
-## 7. Login to Vault
+## 8. Login to Vault
 Run the following commands to login to Vault:
 ```bash
 ./add-to-hosts.sh
 
-export VAULT_ADDR=https://vault-cli.playground.lab
```

Comment:
We don't actually need to maintain two different names for the Vault API, that's just pointless. Let's go with `vault.playground.lab` here @Xelef2000?